.github/workflows/cicd-main.yml

```diff
@@ -5183,15 +5183,263 @@ jobs:
         rm -rf tests/collections/llm/t5_finetune_results/${{ github.run_id }}
 
   L2_NeMo_2_Mixtral_Pretraining:
+      needs: [cicd-test-container-setup]
+      uses: ./.github/workflows/_test_template.yml
+      if: contains(fromJSON(needs.cicd-test-container-setup.outputs.test_to_run), 'L2_NeMo_2_Mixtral_Pretraining') || needs.cicd-test-container-setup.outputs.all == 'true'
+      with:
+        RUNNER: self-hosted-azure
+        SCRIPT: |
+          NVTE_FUSED_ATTN=0 NVTE_FLASH_ATTN=0 python3 tests/collections/llm/megatron_mixtral_pretraining.py \
+          --experiment-dir=/tmp/mixtral_pretrain_results \
+          --data-path=/home/TestData/nlp/megatron_t5/data/pile_val_small_bert_tokenizer_text_document
+
+  L2_NeMo_2_GPT_SFT_TP1PP1_MBS1:
+    needs: [cicd-test-container-setup]
+    uses: ./.github/workflows/_test_template.yml
+    if: contains(fromJSON(needs.cicd-test-container-setup.outputs.test_to_run), 'L2_NeMo_2_GPT_SFT_TP1PP1_MBS1') || needs.cicd-test-container-setup.outputs.all == 'true'
+    with:
+      RUNNER: self-hosted-azure
+      SCRIPT: |
+
+        python tests/collections/llm/gpt_finetuning.py \
+        --restore_path /home/TestData/nemo2_ckpt/llama_68M \
+        --devices 2 \
+        --max_steps 3 \
+        --experiment_dir tests/collections/llm/${{ github.run_id }} \
```

Comment:
i see you are using two different paths for the same
--experiment-dir=/tmp/mixtral_pretrain_results \
--experiment_dir tests/collections/llm/${{ github.run_id }} \
Can we unify the idea & use /tmp only ?

Reply:
Discussed @cuichenx will do this in follow up PR

Reply:
Here's the follow up PR #10859